common/models/user.js

```diff
@@ -210,19 +236,11 @@ module.exports = function(User) {
               err.code = 'LOGIN_FAILED_EMAIL_NOT_VERIFIED';
               return fn(err);
             } else {
-              user.createAccessToken(credentials.ttl, function(err, token) {
-                if (err) return fn(err);
-                if (Array.isArray(include) ? include.indexOf('user') !== -1 : include === 'user') {
-                  // NOTE(bajtos) We can't set token.user here:
-                  //  1. token.user already exists, it's a function injected by
-                  //     "AccessToken belongsTo User" relation
-                  //  2. ModelBaseClass.toJSON() ignores own properties, thus
-                  //     the value won't be included in the HTTP response
-                  // See also loopback#161 and loopback#162
-                  token.__data.user = user;
-                }
-                fn(err, token);
-              });
+              if (user.createAccessToken.length === 2) {
```

Comment:
@raymondfeng Not sure what this means? How can I pass credentials to to createAccessToken?